### PR TITLE
My Capstone Project

### PR DESCRIPTION
**Pull request by lincoln1883**
Milestone 0 - 2, I created the repo and cloned it to my local machine where I:
- Add linters and packages.
- Add assets, images and icons.
- Add boilerplate HTML and CSS.
- Add layout for mobile-first development.
- Add Layout and styles for the intro section.
- Add layout and styling for the events section.
- Add layout and styles for the sponsor's section and add more images.
- Add layout and styles for the footer section.
- Add about page skeleton and edited the page title and footer.
- Add a logo detail and history section to the About page.
- Fix linter errors, pushed and deployed the website.
Milestone 3 + 4, I created a new branch for the Desktop version where I:
- Add layout and styling for the main navigation.
- Adjusted the layout and add styling to the intro section.
- Edited the content and added styling to the feature section.
- Add layout and styling for the sponsors and footer section.
- Add layout and styling for the Top navbar to hold social links and more.
- Add layout and styling to the About page.
- Add interactivity to the website with animations, transitions and links.
- Fixed errors and pushed changes.
Milestone 5 + 6, I created a new branch for the javascript where I:
- Removed the hard-code HTML and dynamically generated the feature section for javascript DOM elements.
- Add documentation for the project including a video describing my work including the challenges.
Milestone 6, Deployment of the website can be viewed here:[Live view](https://lincoln1883.github.io/mod1-capstone-project/)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+.feature-card__category {
+  color: var(--clr-accent);
+  font-weight: 600;
+  font-size: 1rem;
+  font-style: italic;
+}
+
+.featured-description {
+  font-size: 0.9rem;
+}
```
Comment (zieeco) [inline style.css]: - Hey, for the mobile version, to improve the user experience, let's reduce the font size to 0.75rem as the text content's length is too long compared to the original design.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+
+.featured-artist {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  width: 100%;
+  padding: 2rem 1rem;
+}
```
Comment (zieeco) [inline style.css]: - Hey, for the desktop version, let's increase the padding to 3rem 9rem to improve our design to be close to the design template and to improve the user experience on the desktop version.




*design*
Screenshot from 2023-04-20 00-12-16

*your design*
Screenshot from 2023-04-20 00-10-52

File: style.css [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+    flex-wrap: wrap;
+    margin: 30px 0;
+  }
+
+  .events-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 12rem;
+  }
```
Comment (zieeco) [inline style.css]: - Hey, on the desktop version, let's align the text to the center to make it close to the original design.
- let's also add the missing white border on each card.

*design*
Screenshot from 2023-04-20 00-21-40

*your design*
Screenshot from 2023-04-20 00-25-35

File: style.css [hunk 4]
```diff
@@ -0,0 +1,586 @@
...
+  list-style: none;
+  gap: 10px;
+  padding: 0 20px;
+}
+
+.sponsors-item img {
+  width: 8rem;
+  height: 5rem;
+  mix-blend-mode: overlay;
+}
```
Comment (zieeco) [inline style.css]: - Hey, let's add a filter effect on these images to be close to the original design. you can add this  
` filter: grayscale(100%) contrast(30%);`. You can play around with the contrast to adjust it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+				<li class="social__item">
+					<a href="#" class="social__link">Logout</a></li>
+			</ul>
+		</div>
+		<button id="menu-close" class="nav-toggle" type="button" aria-label="toggle navigation">
+				<i class="fa-solid fa-bars"></i>
+		</button>
+		<nav class="nav">
+			<div class="nav__logo">
+				<a href="#"><img class="logo" src="./assets/logos/logo-nav.png" alt="logo"></a>
```
Comment (zieeco) [inline index.html]: - Hey, please, kindly make the logo link to the homepage by making it a live link to the index.html page and not a dead link.

File: README.md [hunk 6]
```diff
@@ -1 +1,199 @@
...
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://lincoln1883.github.io/mod1-capstone-project/)
+
+
+## 🚀 Video Presentation <a name="video-presentation"></a>
+- [Video Presentation](https://www.loom.com/share/59480bb95ab344d19bee283cd3fee8a4)
```
Comment (zieeco) [inline README.md]: - Hey, the video's maximum length should not exceed 5 mins. kindly record another video to ensure you've all the requirements for the video. you can check the project requirement for more details on the video.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+				<li class="sponsors-item">
+					<img src="./assets/images/pepsi.jpeg" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/observer.png" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/flow.png" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/iriefm.jpg" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/jtb-logo.jpg" alt=""></li>
```
Comment (zieeco) [inline index.html]: ### ADD ALTERNATIVE TEXT TO ALL YOUR IMAGES WITHOUT ANY 

- Hey, let's add an alternative text to ALL THE IMAGES without the alt text in the HOME and ABOUT PAGE  to improve the web accessibility of your webpage.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @lincoln1883,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**